travel/src/main/java/com/tx/travel/model/ExchangeRate.java

```diff
@@ -0,0 +1,29 @@
+package com.tx.travel.model;
+
+import jdk.jfr.Unsigned;
+import org.hibernate.validator.constraints.Length;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "exchange_rates",
+        uniqueConstraints = {
+            @UniqueConstraint(columnNames = "currency")
+})
+public class ExchangeRate {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotBlank
+    @NotNull
+    @Length(max = 20)
+    private String currency;
+
+    @NotNull
+    private Float amountInRsd;
```

Comment:
Can we use BigDecimal here?

Reply:
Thanks for the suggestion, I will look into it and implement changes within the next PR.